apps/blog/src/includes/posts.njk

```diff
@@ -1,8 +1,7 @@
 {% import "layouts/postcard.njk" as cardLayouts %}
 
 <section class="my-8">
-  <h2 class="text-center">Featured posts</h2>
+  <h1 class="text-center">PWABuilder Blog</h1>
```

Comment:
There is a discrepancy btwn the tab saying "PWA Builder Blog" (probs from the head of the html) but this is "PWA Builder" which I believe is correct. Just something to update in the index.